deriv.com.conf

```diff
@@ -42,10 +52,10 @@ server {
     }
 
     location /static {
-        add_header Cache-Control "public, max-age=31536000, immutable";
+        add_header Cache-Control $cache_header always;
```

Comment:
I note `always` and I wonder about it, in light of [the documentation](https://nginx.org/en/docs/http/ngx_http_headers_module.html)
We're using our status map to decide what to do here, but there's other status codes we haven't accounted for,
and we'd be defaulting to caching them.

I actually wonder if the lack of `always` has been ignored by all of us on the existing error responses.

Maybe that's *why* this worked and why Gokula chose to do it this way?